src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+  </head>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <body class="block-product">
+    <div
+      data-qa="card"
+      class="block-product__screen screen"
+    >
+      <img
+        class="image"
+        src="images/imac.jpeg"
+        alt="screen"
+      >
+      <h2 class="list">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <h3 class="product">
+        Код товара: 195434
+      </h3>
+      <div
+      class="stars stars--4"
+      >
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <li class="reviews">
+          Отзывов: 5
+        </li>
+      </div>
+      <div class="prices">
+        <li class="price">
+          Цена:
+        </li>
+        <li class="mony">
+          69 999 грн
+        </li>
+      </div>
+      <a
+      href="#КУПИТЬ"
+      data-qa="hover"
+      class="link"
+    >
+        <li>
+            КУПИТЬ
```

Comment:
use css property instead (upper-case)
```suggestion
            Купить
```